complete-application/src/router/index.ts

```diff
@@ -20,6 +20,12 @@ const router = createRouter({
       component: HomeView,
       beforeEnter: routeGuard(false, '/account')
     },
+    {
+      path: '/logged-out',
+      name: 'home',
+      component: HomeView,
+      beforeEnter: routeGuard(false, '/account')
+    },
```

Comment:
This is preventing the UI from rendering.
<img width="493" alt="image" src="https://github.com/FusionAuth/fusionauth-quickstart-javascript-vue-web/assets/49047386/27c5fbb4-9303-4e4f-8de6-a9a5be6fb83c">

I'm not sure why because I haven't used vue router but perhaps try this: https://router.vuejs.org/guide/essentials/redirect-and-alias#Alias

Reply:
I added an alias and that seemed to work! thank you